### PR TITLE
Add activate_game_overlay_to_store and activate_game_overlay_to_user

### LINKED ISSUE #126: Feature Request: Implement more Activate Overlay functions
steamworks-rs is missing these Activate Overlay features from ISteamFriends:

* ActivateGameOverlayToStore
* ActivateGameOverlayToUser

### PR DESCRIPTION
**Pull request by dfabulich**
Fixes #126

### CODE REVIEW

File: src/friends.rs [hunk 1]
```diff
@@ -122,6 +122,27 @@ impl<Manager> Friends<Manager> {
         }
     }
 
+    pub fn activate_game_overlay_to_store(&self, app_id: AppId) {
+        unsafe {
+            sys::SteamAPI_ISteamFriends_ActivateGameOverlayToStore(
+                self.friends,
+                app_id.0,
+                sys::EOverlayToStoreFlag::k_EOverlayToStoreFlag_None,
```
Comment (ceifa) [inline src/friends.rs:130]: would be better if we receive the flag as argument, defaulting can lead to unexpected behaviours

**Comment by dfabulich** [comment]
The build failure looks like it's not in my code. Looks like the Github action was trying to `apt-get update` and it failed to reach its package repository…?

https://github.com/Noxime/steamworks-rs/actions/runs/4977534930/jobs/8912408214#step:4:62

```
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/s/systemd/libudev-dev_249.11-0ubuntu3.9_amd64.deb  503  Service Unavailable [IP: 52.252.75.106 80]
```

**Comment by dfabulich** [comment]
Is this ready to merge?

**Comment by ceifa** [comment]
maybe would be nice if the dialog was an enum too? idk
LGTM

**Pull request merged**